src/App.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  lastPressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    lastPressedKey: '',
+  };
+
+  componentDidMount(): void {
```

Comment:
```suggestion
  componentDidMount() {
```

Reply:
I understand what you're talking about, but if I change `event.key` to `event.code`, the rest of the buttons will be displayed otherwise, so I added a condition to the code so that space is displayed

Reply:
Yes, this too will work, you even could do this check inside getPressedKey.
It does actually, probably should set is as optional, but you've done a great job. You just forgot to deploy your changes🥲

Reply:
You're right, I forgot😅. Already did it